lib/safeEqual.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+function safeEqual (a, b) {
+  if (typeof b !== 'string' || typeof b !== 'string') {
+    return a === b
+  }
+
+  // xor strings for security
+  var mismatch = 0
+  for (var i = 0; i < a.length; ++i) {
```

Comment:
you can probably break early here, as there is no need to keep iterating if there is a mismatch.